lib/glueby/contract/token.rb

```diff
@@ -200,7 +200,8 @@ def to_payload
       def self.parse_from_payload(payload)
         color_id, script_pubkey = payload.unpack('a33a*')
         color_id = Tapyrus::Color::ColorIdentifier.parse_from_payload(color_id)
-        if color_id.type == Tapyrus::Color::TokenTypes::REISSUABLE && !script_pubkey.empty?
+        if color_id.type == Tapyrus::Color::TokenTypes::REISSUABLE
+          raise ArgumentError, 'script_pubkey is not nil.' unless !script_pubkey.empty?
```

Comment:
条件文はこうした方がシンプルだと思います。
```suggestion
          raise ArgumentError, 'script_pubkey is not nil.' if script_pubkey.empty?
```